readthedocs/proxito/views/hosting.py

```diff
@@ -514,6 +512,28 @@ def _v1(self, project, version, build, filename, url, request):
             },
         }
 
+        # Show the subprojects filter on the parent project and subproject
+        if version:
+            # TODO: Remove these queries and try to find a way to get this data
+            # from the resolver, which has already done these queries.
+            # TODO: Replace this fixed filters with the work proposed in
+            # https://github.com/readthedocs/addons/issues/22
+            if project.subprojects.exists():
+                data["addons"]["search"]["filters"].append(
+                    [
+                        "Include subprojects",
+                        f"subprojects:{project.slug}/{version.slug}",
+                    ]
+                )
+            if project.superprojects.exists():
```

Comment:
This can save one query, since a project can't be both, a superproject and subproject.

```suggestion
            elif project.superprojects.exists():
```

Reply:
https://github.com/readthedocs/readthedocs.org/pull/11683